src/main/java/it/gov/pagopa/payhub/auth/controller/AuthzControllerImpl.java

```diff
@@ -99,4 +96,15 @@ public ResponseEntity<UserDTO> createUser(UserDTO user) {
         }
         return ResponseEntity.ok(authzService.createUser(user));
     }
+
+    @Override
+    public ResponseEntity<ClientDTO> registerClient(String organizationIpaCode, CreateClientRequest createClientRequest) {
+        if(organizationAccessMode){
+            return new ResponseEntity<>(HttpStatus.NOT_IMPLEMENTED);
+        }
+        if(!SecurityUtils.hasAdminRole()){
```

Comment:
Only user having admin role on requested organization (current organizationIpaCode) should be allowed.
This way you are checking if it's an Admin for at least one organization

Reply:
there was a misunderstanding, I'm removing that part of the code